src/software/geom/util.cpp

```diff
@@ -693,29 +693,6 @@ Vector closestPointOnLine(const Vector &centre, const Vector &lineA, const Vecto
     return lineB;
 }
 
-namespace
-{
-    std::vector<Vector> lineseg_circle_intersect(const Vector &centre, double radius,
```

Comment:
These were causing a bazel compiler warning since it was unused. Since compiler warnings are now treated as errors, this will cause any build to fail. I could comment it out maybe?

Reply:
I'll agree with Gareth here, let's just delete it for now.